### PR TITLE
Update helm charts to use collector `v0.21.0` - Helm chart `v0.7.0` Release

### PR DESCRIPTION
**Pull request by vasireddy99**
**Description:** 

This PR updates collector version v0.21.0, uses `prometheusremotewriteexporter`with `sigv4auth`, Helm version `v0.7.0`

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** 

image

**Documentation:** <Describe the documentation added.>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: charts/adot-exporter-for-eks-on-ec2/values.yaml [hunk 1]
```diff
@@ -189,7 +189,7 @@ adotCollector:
   image:
     name: "aws-otel-collector"
     repository: "amazon/aws-otel-collector"
-    tag: "v0.17.0"
+    tag: "v0.21.0"
```
Comment (erichsueh3) [inline charts/adot-exporter-for-eks-on-ec2/values.yaml:192]: 21.1?

File: charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/configmap.yaml [hunk 2]
```diff
@@ -42,14 +44,13 @@ data:
...
         namespace: {{ .Values.adotCollector.daemonSet.ampexporters.namespace }}
         endpoint: {{ .Values.adotCollector.daemonSet.ampexporters.endpoint }}
         resource_to_telemetry_conversion: 
-          enabled: {{ .Values.adotCollector.daemonSet.ampexporters.resourcetotel }}
-        aws_auth:
-          region: {{ .Values.adotCollector.daemonSet.ampexporters.region }}
-          service: aps
+          enabled: {{ .Values.adotCollector.daemonSet.ampexporters.resourcetootel }}
+        auth:
+          authenticator: {{ .Values.adotCollector.daemonSet.extensions.sigv4auth }}
```
Comment (erichsueh3) [inline charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/configmap.yaml:53]: is `sigv4auth` a value in values.yaml? In it, it looks like this,
```
extensions:
  healthCheck: ""
  sigv4auth:
    region: ""
```
will `{{ .Values.adotCollector.daemonSet.extensions.sigv4auth }}` return `sigv4auth` as a value?

File: charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/configmap.yaml [hunk 3]
```diff
@@ -13,6 +13,8 @@ data:
   adot-config:  |
     extensions:
       health_check: {{ .Values.adotCollector.daemonSet.extensions.healthCheck }}
+      sigv4auth:
+        region: {{ .Values.awsRegion }}
```
Comment (erichsueh3) [inline charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/configmap.yaml:17]: Why is the region value here different than the region value used in the removed `awsprometheusremtoewrite` config? 
```
aws_auth:
  region: {{ .Values.adotCollector.daemonSet.ampexporters.region }}
```
Are they always the same region? If so, can the above value be removed from `values.yaml`?

Comment (vasireddy99) [reply charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/configmap.yaml:17]: Yes, the above value is removed from values.yaml. and we use `awsRegion` which we pass through at the time of install.

**Review by erichsueh3 — APPROVED** [review]

**Pull request merged**